### DISCUSSION TITLE
Does Scramble support Dingo API projects?

### DISCUSSION THREAD

**Discussion by HaakonFlaaronning** [dedoc/scramble#36] [Q&A] [answered]
Not all auto-generated API documentation solutions supports Dingo - does Scramble?

**Comment by romalytvynenko** [comment, marked as answer]
Hey @HaakonFlaaronning 

Thanks for the question. 

For now, there is no special support for Dingo library from Scramble. Scramble was developed specifically for Laravel and all the conventions in Laravel. 

However, other libraries support can be added via Scramble extensions system: https://scramble.dedoc.co/developers/extensions
